locals.tf

```diff
@@ -54,7 +55,7 @@ locals {
       metadata_http_endpoint               = var.default_nodepool_metadata_http_endpoint
       metadata_http_tokens                 = var.default_nodepool_metadata_http_tokens
       metadata_http_put_response_hop_limit = var.default_nodepool_metadata_http_put_response_hop_limit
-
+      subnets                              = local.worker_vm_subnets
```

Comment:
There is no mention of the new `subnets` parameter in the varaibles.tf file. Need to add it there for both the default node pool and system node pools.

Reply:
This reference of 'subnets' is already defined in EKS module - https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/locals.tf#L68